src/openloco/map/tile.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "../station.h"
```

Comment:
We really should make a light header just for types like `station_id`.

Reply:
I had to do this for Ride in OpenRCT2, you can have something like `station_types.h` which just has forward declarations and typedefs, constants etc.

Reply:
We already have a `types.hpp`. Perhaps we should just extend that?

Reply:
good point